plutus-playground-server/app/PSGenerator.hs

```diff
@@ -290,22 +315,75 @@ multilineString :: Text -> Text -> Text
 multilineString name value =
     "\n\n" <> name <> " :: String\n" <> name <> " = \"\"\"" <> value <> "\"\"\""
 
+jsonExport :: ToJSON a => Text -> a -> Text
+jsonExport name value =
+    multilineString name (T.decodeUtf8 . BSL.toStrict $ JSON.encode value)
+
 psModule :: Text -> Text -> Text
 psModule name body = "module " <> name <> " where" <> body
 
+------------------------------------------------------------
 writeUsecases :: FilePath -> IO ()
 writeUsecases outputDir = do
     let usecases =
             multilineString "vesting" vesting <> multilineString "game" game <>
-            multilineString "crowdfunding" crowdfunding <>
+            multilineString "crowdFunding" crowdFunding <>
             multilineString "errorHandling" errorHandling <>
-            multilineString "starter" starter
+            multilineString "starter" starter <>
+            jsonExport "contractDemos" contractDemos
         usecasesModule = psModule "Playground.Usecases" usecases
     BS.writeFile
         (outputDir </> "Playground" </> "Usecases.purs")
         (T.encodeUtf8 usecasesModule)
-    putStrLn outputDir
 
+------------------------------------------------------------
+writeTestData :: FilePath -> IO ()
+writeTestData outputDir = do
+    let ContractDemo {contractDemoContext, contractDemoSimulations} =
+            gameContractDemo
+    BSL.writeFile
+        (outputDir </> "compilation_response.json")
+        (JSON.encodePretty contractDemoContext)
+    void $
+        itraverse
+            (\index ->
+                 writeSimulation
+                     (outputDir </> "evaluation_response" <> show index <>
+                      ".json"))
+            contractDemoSimulations
+
+writeSimulation :: FilePath -> Simulation -> IO ()
+writeSimulation filename simulation = do
+    result <- runExceptT $ runSimulation simulation
+    case result of
+        Left err   -> fail $ "Error evaluating simulation: " <> show err
+        Right json -> BSL.writeFile filename json
+
+runSimulation ::
+       (MonadMask m, MonadError PlaygroundError m, MonadIO m)
+    => Simulation
+    -> m BSL.ByteString
+runSimulation Simulation {simulationActions, simulationWallets} = do
+    let maxInterpretationTime :: Microsecond
+        maxInterpretationTime = fromMicroseconds (80 * 1000 * 1000)
```

Comment:
you can use `80_000_000` these days

Reply:
It needs the `NumericUnderscores` extension, which sadly doesn't play well with hindent. :-(

https://github.com/chrisdone/hindent/issues/542

Reply:
Ah, I think I can sidestep the whole thing by doing this instead:

```haskell
maxInterpretationTime :: Second
maxInterpretationTime = fromIntegral 80
```